src/pages/CreatePool/index.tsx

```diff
@@ -147,17 +148,18 @@ export default function CreatePool(): React.ReactElement {
   }
 
   const poolNameError = poolName.length > 32
-  const poolSymbolError = poolSymbol.length > 32
+  const poolSymbolError = poolSymbol.length > 32 || poolSymbol.includes(" ")
   const aParameterError =
     !isValidNumber(aParameter) || parseFloat(aParameter) < 1
 
   const feeError =
-    !isValidNumber(fee) || parseFloat(fee) > 1 || parseFloat(fee) < 0.04
+    !isValidNumber(fee) || parseFloat(fee) > 1 || parseFloat(fee) < 0.01
 
   useEffect(() => {
     const getBasePoolLPTokenAddrs = async () => {
       if (poolRegistry) {
-        const basePoolName = poolType === PoolType.UsdMeta ? "USDv2" : "BTCv2"
+        const basePoolName =
```

Comment:
note this is only applicable for mainnet, will need to update as new chains are supported

Reply:
TODO in this PR - disable route if chain != mainnet (or hardhat)